frostsnapp/lib/theme.dart

```diff
@@ -39,15 +41,15 @@ final ThemeData frostsnappTheme = ThemeData(
   iconTheme: IconThemeData(color: textSecondaryColor),
   checkboxTheme: CheckboxThemeData(
     checkColor:
-        MaterialStateProperty.resolveWith<Color>((Set<MaterialState> states) {
-      if (states.contains(MaterialState.selected)) {
+        WidgetStateProperty.resolveWith<Color>((Set<WidgetState> states) {
```

Comment:
Note: upgrade to recent flutter version.

After `flutter upgrade` some of the background colours stopped rendering so I fixed them in 383703cb0a0a8ed5faece81e76cb4b503dcc4823